prosjekt1/magnus/styles.css

```diff
@@ -1 +1,27 @@
 
+body {
+    text-align: center;
+}
+
+ol {
+    width: 200px;
+    margin: 0 auto;
+    margin-bottom: 20px;
+}
+
+img{ 
+    width: 350px;
+    height: 200px;
+}
+
+.Albumer{
```

Comment:
Store bokstaver  i classname? Vet ikke hva som er presedens

Reply:
thanks for good feedback!